client/src/components/CommonComponents/Tabs/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import useStyles from './style';
+
+export default function DisabledTabs() {
```

Comment:
Why is it called `DisabledTabs`? why not just `Tabs`?

Reply:
ah that's a mistake, it was tabs

Reply:
but then the thing is Tabs has been declared on Material ui so js won't understand since we have Tabs component has been declared before the function that's why but I have changed it to another name